applications/GiD/tapisjob_app.sh

```diff
@@ -0,0 +1,211 @@
+set -x
```

Comment:
Is this file being used?

Reply:
No, it's using the one on this location --> corral/tacc/aci/CEP/applications/v3/interactive-template/interactive.zip
I will remove this one.